docs/changelogs/v2.10.0.md

```diff
@@ -0,0 +1,125 @@
+
+
+## Changelog
+
+> [!NOTE]
+> This is a Mainline Coder release. We advise enterprise customers without a staging environment to install our [latest stable release](https://github.com/coder/coder/releases/latest) while we refine this version. Learn more about how we release in our [docs](../install/releases.md).
+
+
+### BREAKING CHANGES
+
+- Removed `max_ttl` from templates (#12644) (@Emyrk)
```

Comment:
Can you expand this a bit, my thoughts are the following:

- Explain what `max_ttl` is rendered as in the UI, as this is the API term
- Explain the impact to existing templates who have this applied
- Explain what feature(s) people can use instead of max_ttl to achieve a similar outcome
- Explain our rationale for removing (e.g. it has been deprecated since x)

Reply:
> Explain the impact to existing templates who have this applied

Plus for existing workspaces